generator/.DevConfigs/3aa6313d-9526-40ba-b09c-e046e0d4ef2f.json

```diff
@@ -0,0 +1,13 @@
+{
+  "core": {
+    "changeLogMessages": [
+      "Implement multi-auth SEP with authentication scheme preference and SigV4a region set configuration",
```

Comment:
For the final changelog, we will want to refactor these (SEP doesn't mean anything to customers - and even internal terms like auth resolver are not relevant). 

We should focus on the actual impact - e.g. ability to set SigV4A explicitly.